src/components/WorkOrders/SummaryUpdateJob.js

```diff
@@ -0,0 +1,70 @@
+import PropTypes from 'prop-types'
+import { useForm } from 'react-hook-form'
+import { Button } from '../Form'
+
+const SummaryUpdateJob = ({
+  reference,
+  onJobSubmit,
+  task,
+  sorCodesCollection,
+  changeStep,
+}) => {
+  const { handleSubmit } = useForm({})
+
+  const tableData = (data) => {
+    return data.map((t, index) => (
+      <tr className={`task-${index} govuk-table__row`}>
+        <th scope="row" className={` sor-code-${index} govuk-table__header`}>
+          {t.code}
+        </th>
+        <td className={`quantity-${index} govuk-table__cell`}>{t.quantity}</td>
+        <td className={`edit-${index} govuk-table__cell`}>
+          <a onClick={changeStep}>Edit</a>
```

Comment:
I think we might need an `href="#"` on this link, otherwise the user doesn't get the normal link cursor when they hover.

Reply:
I will add `href="#" ` to SummaryCloseJob as well then
good catch ;)